tests/conftest.py

```diff
@@ -18,3 +21,25 @@ def mock_ezfuncs(mocker):
 @pytest.fixture
 def mock_database_access(mock_ezfuncs):
     return {"cursor": mock_ezfuncs.get_connection().cursor(), "connection": mock_ezfuncs.get_connection()}
+
+
+def pytest_addoption(parser):
```

Comment:
Neat, but dear lord why is this a magic function name and not a decorator off the pytest module or something like that that's consistent with the rest of their API. Not your code, I know. Bleck.